code/game/objects/effects/spawners/lootdrop.dm

```diff
@@ -108,13 +110,51 @@
 				/obj/item/weapon/storage/box/donkpockets = 1,
 				/obj/item/weapon/storage/box/lights/mixed = 3,
 				/obj/item/weapon/storage/fancy/cigarettes/dromedaryco = 1,
-				/obj/item/weapon/storage/toolbox/mechanical = 1,
-				/obj/item/weapon/screwdriver = 3,
+				/obj/item/weapon/storage/toolbox/mechanical = 1,,
 				/obj/item/weapon/tank/emergency_oxygen = 2,
 				/obj/item/weapon/vending_refill/cola = 1,
-				/obj/item/weapon/weldingtool = 3,
```

Comment:
did you actually add these back anywhere? Because I don't really support removing anything from the system.